java/ql/src/semmle/code/java/Expr.qll

```diff
@@ -469,6 +479,12 @@ class ArrayInit extends Expr, @arrayinit {
   /** Gets the initializer occurring at the specified (zero-based) position. */
   Expr getInit(int index) { result = this.getAnInit() and result.getIndex() = index }
 
+  /**
+   * Gets the number of expressions in this initializer, i.e. the size the
```

Comment:
Minor nit: We try to avoid abbreviations like this in the qldoc.
```suggestion
   * Gets the number of expressions in this initializer, that is, the size the
```